common-lib/src/main/scala/com/gu/mediaservice/lib/collections/CollectionsManager.scala

```diff
@@ -0,0 +1,25 @@
+package com.gu.mediaservice.lib.collections
+
+import java.net.URLDecoder.decode
+import java.net.URLEncoder.encode
+
+import com.gu.mediaservice.model.Collection
+
+object CollectionsManager {
+  val delimiter = "/"
+  val enc = "UTF-8"
+  def decodePathBit(s: String) = decode(s, enc)
+  def encodePathBit(s: String) = encode(s, enc)
+
+  def stringToPath(s: String) = s.split(delimiter).map(decodePathBit).toList
+  def pathToString(path: List[String]) = path.map(encodePathBit).mkString(delimiter)
+
+  def add(collection: Collection, collections: List[Collection]): List[Collection] =
+    collection :: collections.filter(col => col.path != collection.path)
```

Comment:
Easier to read manually (it's in order), and that should mean the server response is ordered as well, which makes it easier to browse visually, and potentially more efficient to aggregate on the client.